cmd/soroban-cli/src/commands/completion.rs

```diff
@@ -26,6 +26,6 @@ pub struct Cmd {
 impl Cmd {
     pub fn run(&self) {
         let cmd = &mut Root::command();
-        generate(self.shell, cmd, env!("CARGO_BIN_NAME"), &mut io::stdout());
+        generate(self.shell, cmd, "soroban", &mut io::stdout());
```

Comment:
This env var wasn't present for a library build so this was failing.  If this is important I can make the 'main.rs` pass this value.